tests/dream/io/geant4_test.py

```diff
@@ -176,18 +176,20 @@ def test_load_geant4_csv_sans_has_expected_coords(file):
     assert "position" in sans.coords
 
 
-def test_geant4_in_pipeline(file_path, file):
-    pipeline = LoadGeant4Workflow()
-    pipeline[Filename[SampleRun]] = file_path
-    pipeline[NeXusDetectorName] = NeXusDetectorName("mantle")
-    pipeline[NeXusComponent[snx.NXsample, SampleRun]] = sc.DataGroup(
+def test_geant4_in_workflow(file_path, file):
+    wf = DreamGenericWorkflow()
+    for provider in geant4_providers:
+        wf.insert(provider)
+    wf[Filename[SampleRun]] = file_path
+    wf[NeXusDetectorName] = NeXusDetectorName("mantle")
+    wf[NeXusComponent[snx.NXsample, SampleRun]] = sc.DataGroup(
         position=sc.vector([0.0, 0.0, 0.0], unit="mm")
     )
-    pipeline[NeXusComponent[snx.NXsource, SampleRun]] = sc.DataGroup(
+    wf[NeXusComponent[snx.NXsource, SampleRun]] = sc.DataGroup(
         position=sc.vector([-3.478, 0.0, -76550], unit="mm")
     )
```

Comment:
Not sure why this is not using the Geant4 workflows, where these things are defined already?